vars/mavenNode.groovy

```diff
@@ -5,6 +5,19 @@ def call(Map parameters = [:], body) {
     def defaultLabel = buildId('maven')
     def label = parameters.get('label', defaultLabel)
 
+    def file = new File("/var/lib/jenkins/nodes")
+    
+    if(!file.exists()){
+        error "file or directory /var/lib/jenkins/nodes doesnot exist"
+    } else {
+        if(!file.isDirectory()){
+            error "/var/lib/jenkins/nodes exists, but it is not a directory"
+        } else {
+            println "/var/lib/jenkins/nodes exists and it's a directory"
```

Comment:
ast @sthaha mentioned its betetr to extract `/var/lib/jenkins/nodes` into a variable.